hazelcast/src/test/java/com/hazelcast/map/SizeEstimatorTest.java

```diff
@@ -57,7 +57,7 @@ public void testSinglePut() throws InterruptedException {
 
     @Test
     public void testExactHeapCostAfterUpdateWithMultipleBackupNodes() throws InterruptedException {
-        long expectedPerEntryHeapCost = 172L;
+        long expectedPerEntryHeapCost = 156L;
```

Comment:
Does this not depend on the JVM settings? E.g. if you use compressed or non compressed oops then removing a pointer will reduce size with either 4 or 8 bytes.